src/main/java/org/isf/examination/rest/ExaminationController.java

```diff
@@ -268,4 +208,42 @@ public ResponseEntity<List<PatientExaminationDTO>> getByPatientId(@PathVariable
+        	throw new OHAPIException(new OHExceptionMessage("The weight should be between"+ExaminationParameters.WEIGHT_MIN+" and "+ExaminationParameters.WEIGHT_MAX));
+        }
+        if (newPatientExamination.getPex_ap_min() < ExaminationParameters.AP_MIN_INIT) {
+        	throw new OHAPIException(new OHExceptionMessage("The minimum blood pressure must be at least "+ExaminationParameters.AP_MIN_INIT));
+        }
+        if (newPatientExamination.getPex_ap_min() > newPatientExamination.getPex_ap_max() ) {
+        	throw new OHAPIException(new OHExceptionMessage("The minimum blood pressure must be lower than "+ExaminationParameters.AP_MAX_INIT));
+        }
+        if (newPatientExamination.getPex_ap_max() > ExaminationParameters.AP_MAX_INIT) {
+        	throw new OHAPIException(new OHExceptionMessage("The maximum blood pressure must be lower than "+ExaminationParameters.AP_MAX_INIT));
+        }
+        if (newPatientExamination.getPex_hr() < ExaminationParameters.HR_MIN || newPatientExamination.getPex_hr() > ExaminationParameters.HR_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("Heart rate should be between "+ExaminationParameters.HR_MIN +" and "+ExaminationParameters.HR_MAX));
+        }
+        if (newPatientExamination.getPex_temp() < ExaminationParameters.TEMP_MIN || newPatientExamination.getPex_temp() > ExaminationParameters.TEMP_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("The temperature should be between "+ExaminationParameters.TEMP_MIN +" and "+ExaminationParameters.TEMP_MAX));
+        }
+        if (newPatientExamination.getPex_sat() < ExaminationParameters.SAT_MIN || newPatientExamination.getPex_temp() > ExaminationParameters.SAT_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("The saturation should be between "+ExaminationParameters.SAT_MIN+" and "+ExaminationParameters.SAT_MAX));
+        }
+        if (newPatientExamination.getPex_hgt() < ExaminationParameters.HGT_MIN || newPatientExamination.getPex_temp() > ExaminationParameters.HGT_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("HGT should be between "+ExaminationParameters.HGT_MIN+" and "+ExaminationParameters.HGT_MAX));
+        }
+        if (newPatientExamination.getPex_rr() < ExaminationParameters.RR_MIN || newPatientExamination.getPex_rr() > ExaminationParameters.RR_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("Respiratory rate should be between "+ExaminationParameters.RR_MIN+" and "+ExaminationParameters.RR_MAX));
+        }
+        if (newPatientExamination.getPex_diuresis() < ExaminationParameters.DIURESIS_MIN || newPatientExamination.getPex_diuresis() > ExaminationParameters.DIURESIS_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("Diuresis should be between "+ExaminationParameters.DIURESIS_MIN+" and "+ExaminationParameters.DIURESIS_MAX));
+        }
```

Comment:
In general, except height and weight that are always compulsory, the other parameters should be checked only if not null (means not observed)

Reply:
okay